packages/xo-server/src/xo.js

```diff
@@ -20,6 +21,8 @@ export default class Xo extends EventEmitter {
   constructor(config) {
     super()
 
+    this.emitAsync = emitAsync
```

Comment:
You don't need this, you can `emitAsync.call(this)` instead when emitting the event.

It's a bit better because it improves code locality :slightly_smiling_face: